sisbasekt/api.md

```diff
@@ -1,3 +1,38 @@
 # sisbase-api
 `Type: Subproject`  
 Subproject of: [`SisbaseKT`](../sisbasekt.md)
+
+A collection of interfaces representing the Discord API Spec.  
+Implemented by `Backends`.  
+
+## Dependencies
+[types](types.md)  
+
+## Project Structure
+`org.sisbase.sisbase-api`  
+Base package  
+
+`org.sisbase.sisbase-api.rest`  
+High level wrapper for Discord's Rest API  
+
```

Comment:
What is included on the "High level wrapper". Please define the features included on the API.